Update device patch files that error'ed when applied

fidoedidoe opened the pull request:
As originally highlighted in an XDA post ([link](https://forum.xda-developers.com/showpost.php?p=79806889&postcount=373)) these changes allow the the device specific patches to applied without error (at least in my environment - Ubuntu 19.04) and for the build process go on an complete.

fidoedidoe committed:
Update device patch files that errored when applied

fidoedidoe committed:
Update device patch files that errored when applied

Spookcity commented:
Thanks for pointing this out. I just replied to your post on XDA. When I have a little time I'll try to do a clean apply of these patches in their current state and see if I too have problems. Then I'll revisit this. Thanks!

fidoedidoe committed:
Merge remote-tracking branch 'upstream/P' into P

Spookcity committed:
optional_patches:Remove LiveDisplay patches

Spookcity committed:
Set ro.config.small_battery to true

Effectively this will trigger device-idle in 15
minutes instead of 30 minutes (Inactive-timeout and
Idle-after-inactive-timeout)

as seen
here:https://github.com/rINanDO/android_device_samsung_galaxys2-common/commit/d62842c059b603f5ce3a8ee51b082122a00a0817

Spookcity committed:
Add "TARGET_LEGACY_HW_DISK_ENCRYPTION := true" flag

I am not sure this helps anything.Especially without the (#)commented
out flag,but this results in a build error-missing file. This may be a
Qcomm only thing anyway.Not sure.

fidoedidoe committed:
Merge remote-tracking branch 'origin/P' into P
